packages/editor/src/components/post-visibility/style.scss

```diff
@@ -1,19 +1,34 @@
 .edit-post-post-visibility__dialog {
+	.editor-post-visibility__dialog-fieldset {
+		padding: 3px;
```

Comment:
Would it break the visuals if you replaces this with the new 4px `$grid-size-small` variable? I'm slowly trying to roll out the beginnings of a grid, and it'd be nice to use the new units wherever we use 4, 8 or 16px widths.